hub/scribe/service.py

```diff
@@ -1163,6 +1183,28 @@ def _get_cumulative_update_ops(self, height: int):
                  if claim_hash not in self.abandoned_claims}
             )
 
+        # update the reposted counts
+        reposted_to_check = [(claim_hash,) for claim_hash, delta in self.reposted_count_delta.items() if delta != 0]
+        existing_repost_counts = {}
+        if reposted_to_check:
+            existing_repost_counts.update({
+                claim_hash: v.reposted_count
+                for claim_hash, v in zip(self.reposted_count_delta, self.db.prefix_db.reposted_count.multi_get(
+                    reposted_to_check
+                )) if v is not None
+            })
+        if existing_repost_counts:
+            self.db.prefix_db.multi_delete([
+                self.db.prefix_db.reposted_count.pack_item(claim_hash, count)
+                for claim_hash, count in existing_repost_counts.items()
+            ])
```

Comment:
I'm kind of surprised to see this. Is it not possible to put & overwrite the existing values? There must be something in the framework that requires doing it this way that I am not aware of.

Reply:
Existing values can't be put on top of without first deleting them - nor can non-existent values be deleted. This allows easily undoing a series of changes since you can reverse their order and flip the deletes into puts and the puts into deletes.